drivers/input/Kconfig

```diff
@@ -45,13 +45,58 @@ config INPUT_TOUCHSCREEN_NPOLLWAITERS
 
 endif # INPUT_TOUCHSCREEN
 
+config INPUT_KEYBOARD
+	bool
+	select MM_CIRCBUF
+	default n
+
+if INPUT_KEYBOARD
+
+config INPUT_KEYBOARD_BUFFSIZE
+	int "Number keyboard data buffer"
+	default 64
+	---help---
+		Maximum number of keyboard data buffer
+
+endif # INPUT_KEYBOARD
+
 config INPUT_UINPUT
-	bool "Enable uinput"
-	select INPUT_TOUCHSCREEN
+	bool
```

Comment:
```suggestion
	bool "Enable uinput"
```

Reply:
When using UINPUT_KEYBOARD/BUTTON , it will be selected.
It is used to select the source file in the make.defs.

Reply:
ok.